src/effects/native/flangereffect.cpp

```diff
@@ -28,26 +31,60 @@ EffectManifest FlangerEffect::getManifest() {
     period->setId("period");
     period->setName(QObject::tr("Period"));
     period->setDescription(QObject::tr("Controls the period of the LFO (low frequency oscillator)\n"
-        "1/4 - 4 beats rounded to 1/2 beat if tempo is detected (decks and samplers) \n"
-        "0.05 - 4 seconds if no tempo is detected (mic & aux inputs, master mix)"));
+        "1/4 - 20 beats rounded to 1/2 beat if tempo is detected (decks and samplers) \n"
+        "0.05 - 20 seconds if no tempo is detected (mic & aux inputs, master mix)"));
     period->setControlHint(EffectManifestParameter::ControlHint::KNOB_LINEAR);
     period->setSemanticHint(EffectManifestParameter::SemanticHint::UNKNOWN);
     period->setUnitsHint(EffectManifestParameter::UnitsHint::BEATS);
     period->setMinimum(0.00);
-    period->setMaximum(4.00);
+    period->setMaximum(20.00);
     period->setDefault(1.00);
 
-    EffectManifestParameter* depth = manifest.addParameter();
-    depth->setId("depth");
-    depth->setName(QObject::tr("Depth"));
-    depth->setDescription(QObject::tr("Controls the intensity of the effect."));
-    depth->setControlHint(EffectManifestParameter::ControlHint::KNOB_LINEAR);
-    depth->setSemanticHint(EffectManifestParameter::SemanticHint::UNKNOWN);
-    depth->setUnitsHint(EffectManifestParameter::UnitsHint::UNKNOWN);
-    depth->setDefaultLinkType(EffectManifestParameter::LinkType::LINKED);
-    depth->setDefault(1.0);
-    depth->setMinimum(0.0);
-    depth->setMaximum(1.0);
+    EffectManifestParameter* width = manifest.addParameter();
+    width->setId("width");
+    width->setName(QObject::tr("Width"));
+    width->setDescription(QObject::tr("Controls the delay amplitude of the LFO (low frequency oscillator)."));
+    width->setControlHint(EffectManifestParameter::ControlHint::KNOB_LINEAR);
+    width->setSemanticHint(EffectManifestParameter::SemanticHint::UNKNOWN);
+    width->setUnitsHint(EffectManifestParameter::UnitsHint::UNKNOWN);
+    width->setDefaultLinkType(EffectManifestParameter::LinkType::LINKED);
```

Comment:
This is not suitable as a parameter to link to the metaknob by default because there is no way to make the effect not change the sound by manipulating this parameter. When the metaknob is fully left, the effect should not be heard.

Reply:
That is no requirement for the super knob, because this use case is issued with the dry /wet knob. Mapping super to the Mix will just duplicate that behaviour.

Reply:
The Mix parameter is redundant anyway. I have kept it because it was the old depth knob. Can we remove it?

Reply:
It is a requirement that every effect can make no audible change with the default metaknob linking, otherwise there is no way to bring in the effect without a sudden change without abusing the chain dry/wet knob which manipulates all the effects of the chain together.

Reply:
What do you propose?

Reply:
I propose keeping the metaknob linked to the Mix knob by default. Linking the metaknob to the Width parameter is a valid use case, but I don't think it should be the default.

Reply:
Could we make it so the effect does not make an audible change when Width is 0? If so, let's remove the Mix parameter and leave Width linked to the metaknob.

Reply:
Linking to width does not work, because width sets the width that is cycled around the manual position.